### PR TITLE
Fix version `on_flushed`

### PR DESCRIPTION
**Pull request by huachaohuang**
/cc @w41ter

### CODE REVIEW

**Review by w41ter — APPROVED** [review]

File: src/lib.rs [hunk 1]
```diff
@@ -80,9 +82,11 @@ mod tests {
...
             page_size: 64,
             page_chain_length: 2,
-            ..Default::default()
+            page_store: PageStoreOptions {
+                write_buffer_capacity: 1 << 20,
+            },
         };
-        let table = Table::open(path, options).await.unwrap();
+        let table = Table::open(&path, options).await.unwrap();
         const N: u64 = 1024;
```
Comment (huachaohuang) [inline src/lib.rs:90]: There are still some problems if we increase this to `102400`:

```
thread 'tokio-runtime-worker' panicked at 'removal index (is 0) should be < len (is 0)', src/page_store/version.rs:389:34
stack backtrace:
   0: rust_begin_unwind
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:556:5
   1: core::panicking::panic_fmt
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/panicking.rs:142:14
   2: alloc::vec::Vec<T,A>::remove::assert_failed
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/alloc/src/vec/mod.rs:1441:13
   3: alloc::vec::Vec<T,A>::remove
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/alloc/src/vec/mod.rs:1446:13
   4: photondb::page_store::version::BufferSet::on_flushed
             at ./src/page_store/version.rs:389:26
   5: photondb::page_store::jobs::flush::FlushCtx<E>::flush::{{closure}}
             at ./src/page_store/jobs/flush.rs:113:9
   6: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/future/mod.rs:91:19
   7: photondb::page_store::jobs::flush::FlushCtx<E>::run::{{closure}}
             at ./src/page_store/jobs/flush.rs:74:62
   8: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/future/mod.rs:91:19
   9: tokio::runtime::task::core::CoreStage<T>::poll::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/core.rs:184:17
  10: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/loom/std/unsafe_cell.rs:14:9
  11: tokio::runtime::task::core::CoreStage<T>::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/core.rs:174:13
  12: tokio::runtime::task::harness::poll_future::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:480:19
  13: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/panic/unwind_safe.rs:271:9
  14: std::panicking::try::do_call
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:464:40
  15: ___rust_try
  16: std::panicking::try
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:428:19
  17: std::panic::catch_unwind
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panic.rs:137:14
  18: tokio::runtime::task::harness::poll_future
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:468:18
  19: tokio::runtime::task::harness::Harness<T,S>::poll_inner
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:104:27
  20: tokio::runtime::task::harness::Harness<T,S>::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:57:15
  21: tokio::runtime::task::raw::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/raw.rs:194:5
  22: tokio::runtime::task::raw::RawTask::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/raw.rs:134:18
  23: tokio::runtime::task::LocalNotified<S>::run
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/mod.rs:385:9
  24: tokio::runtime::scheduler::multi_thread::worker::Context::run_task::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:421:13
  25: tokio::coop::with_budget::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/coop.rs:102:9
  26: std::thread::local::LocalKey<T>::try_with
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/thread/local.rs:446:16
  27: std::thread::local::LocalKey<T>::with
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/thread/local.rs:422:9
  28: tokio::coop::with_budget
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/coop.rs:95:5
  29: tokio::coop::budget
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/coop.rs:72:5
  30: tokio::runtime::scheduler::multi_thread::worker::Context::run_task
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:420:9
  31: tokio::runtime::scheduler::multi_thread::worker::Context::run
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:387:24
  32: tokio::runtime::scheduler::multi_thread::worker::run::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:372:17
  33: tokio::macros::scoped_tls::ScopedKey<T>::set
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/macros/scoped_tls.rs:61:9
  34: tokio::runtime::scheduler::multi_thread::worker::run
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:369:5
  35: tokio::runtime::scheduler::multi_thread::worker::Launch::launch::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:348:45
  36: <tokio::runtime::blocking::task::BlockingTask<T> as core::future::future::Future>::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/blocking/task.rs:42:21
  37: tokio::runtime::task::core::CoreStage<T>::poll::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/core.rs:184:17
  38: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/loom/std/unsafe_cell.rs:14:9
  39: tokio::runtime::task::core::CoreStage<T>::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/core.rs:174:13
  40: tokio::runtime::task::harness::poll_future::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:480:19
  41: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/panic/unwind_safe.rs:271:9
  42: std::panicking::try::do_call
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:464:40
  43: ___rust_try
  44: std::panicking::try
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:428:19
  45: std::panic::catch_unwind
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panic.rs:137:14
  46: tokio::runtime::task::harness::poll_future
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:468:18
  47: tokio::runtime::task::harness::Harness<T,S>::poll_inner
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:104:27
  48: tokio::runtime::task::harness::Harness<T,S>::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:57:15
  49: tokio::runtime::task::raw::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/raw.rs:194:5
  50: tokio::runtime::task::raw::RawTask::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/raw.rs:134:18
  51: tokio::runtime::task::UnownedTask<S>::run
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/mod.rs:422:9
  52: tokio::runtime::blocking::pool::Task::run
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/blocking/pool.rs:111:9
  53: tokio::runtime::blocking::pool::Inner::run
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/blocking/pool.rs:346:17
  54: tokio::runtime::blocking::pool::Spawner::spawn_thread::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/blocking/pool.rs:321:13
```

/cc @w41ter

File: src/page_store/version.rs [hunk 2]
```diff
@@ -385,7 +385,7 @@ impl BufferSet {
```
Comment (huachaohuang) [inline src/page_store/version.rs:385]: The above assertion also fails sometimes.

Comment (huachaohuang) [reply src/page_store/version.rs:385]:
```
thread 'tokio-runtime-worker' panicked at 'assertion failed: `(left == right)`
  left: `863`,
 right: `864`', src/page_store/version.rs:384:9
stack backtrace:
   0: rust_begin_unwind
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:556:5
   1: core::panicking::panic_fmt
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/panicking.rs:142:14
   2: core::panicking::assert_failed_inner
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/panicking.rs:224:17
   3: core::panicking::assert_failed
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/panicking.rs:181:5
   4: photondb::page_store::version::BufferSet::on_flushed
             at ./src/page_store/version.rs:384:9
   5: photondb::page_store::jobs::flush::FlushCtx<E>::flush::{{closure}}
             at ./src/page_store/jobs/flush.rs:113:9
   6: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/future/mod.rs:91:19
   7: photondb::page_store::jobs::flush::FlushCtx<E>::run::{{closure}}
             at ./src/page_store/jobs/flush.rs:74:62
   8: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/future/mod.rs:91:19
   9: tokio::runtime::task::core::CoreStage<T>::poll::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/core.rs:184:17
  10: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/loom/std/unsafe_cell.rs:14:9
  11: tokio::runtime::task::core::CoreStage<T>::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/core.rs:174:13
  12: tokio::runtime::task::harness::poll_future::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:480:19
  13: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/panic/unwind_safe.rs:271:9
  14: std::panicking::try::do_call
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:464:40
  15: ___rust_try
  16: std::panicking::try
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:428:19
  17: std::panic::catch_unwind
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panic.rs:137:14
  18: tokio::runtime::task::harness::poll_future
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:468:18
  19: tokio::runtime::task::harness::Harness<T,S>::poll_inner
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:104:27
  20: tokio::runtime::task::harness::Harness<T,S>::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:57:15
  21: tokio::runtime::task::raw::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/raw.rs:194:5
  22: tokio::runtime::task::raw::RawTask::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/raw.rs:134:18
  23: tokio::runtime::task::LocalNotified<S>::run
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/mod.rs:385:9
  24: tokio::runtime::scheduler::multi_thread::worker::Context::run_task::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:421:13
  25: tokio::coop::with_budget::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/coop.rs:102:9
  26: std::thread::local::LocalKey<T>::try_with
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/thread/local.rs:446:16
  27: std::thread::local::LocalKey<T>::with
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/thread/local.rs:422:9
  28: tokio::coop::with_budget
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/coop.rs:95:5
  29: tokio::coop::budget
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/coop.rs:72:5
  30: tokio::runtime::scheduler::multi_thread::worker::Context::run_task
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:420:9
  31: tokio::runtime::scheduler::multi_thread::worker::Context::run
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:387:24
  32: tokio::runtime::scheduler::multi_thread::worker::run::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:372:17
  33: tokio::macros::scoped_tls::ScopedKey<T>::set
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/macros/scoped_tls.rs:61:9
  34: tokio::runtime::scheduler::multi_thread::worker::run
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:369:5
  35: tokio::runtime::scheduler::multi_thread::worker::Launch::launch::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/scheduler/multi_thread/worker.rs:348:45
  36: <tokio::runtime::blocking::task::BlockingTask<T> as core::future::future::Future>::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/blocking/task.rs:42:21
  37: tokio::runtime::task::core::CoreStage<T>::poll::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/core.rs:184:17
  38: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/loom/std/unsafe_cell.rs:14:9
  39: tokio::runtime::task::core::CoreStage<T>::poll
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/core.rs:174:13
  40: tokio::runtime::task::harness::poll_future::{{closure}}
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:480:19
  41: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/core/src/panic/unwind_safe.rs:271:9
  42: std::panicking::try::do_call
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:464:40
  43: ___rust_try
  44: std::panicking::try
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panicking.rs:428:19
  45: std::panic::catch_unwind
             at /rustc/8ce3204af9463db3192ea1eb31c45c2f6d4b5ae6/library/std/src/panic.rs:137:14
  46: tokio::runtime::task::harness::poll_future
             at /Users/richard/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/tokio-1.21.2/src/runtime/task/harness.rs:468:18
```

File: src/lib.rs (hunk 1 above)
Comment (w41ter) [reply src/lib.rs:90]: I can't reproduce it on my laptop, are there any other changes?

Comment (w41ter) [reply src/lib.rs:90]: Oh i see, the output is not captured.

Comment (w41ter) [reply src/lib.rs:90]: I still can't reproduce this case.

**Comment by huachaohuang** [comment]
Closes in favor of https://github.com/photondb/photondb/pull/203

**Pull request closed without merging**